bin/python-stripped-env

```diff
@@ -43,14 +43,14 @@ function debug {
 debug "which python: $(which python)"
 
 # unset all $LD_* environment variables
-for env_var in $(env | egrep '^LD_' | cut -f1 -d'='); do
+for env_var in $(env | egrep --color=never '^LD_' | cut -f1 -d'='); do
```

Comment:
just unset GREP_OPTIONS, much safer and future proof

Reply:
Nope, from the man page:

```
       GREP_OPTIONS
              This variable specifies default options to be placed in front of any explicit options.  As this causes problems when writing portable scripts, this feature will be removed
              in a future release of grep, and grep warns if it is used.  Please use an alias or script instead.
```

Reply:
so unsetting it is actually recommended...

Reply:
ok